pilot/util/disk.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Giampaolo Rodola, g.rodola@gmail.com
+# - Daniel Drizhuk, d.drizhuk@gmail.com, 2017
+
+import os
+import collections
+
+_ntuple_diskusage = collections.namedtuple('usage', 'total used free')
+
+if hasattr(os, 'statvfs'):  # POSIX
+    def disk_usage(path):
+        st = os.statvfs(path)
+        free = st.f_bavail * st.f_frsize
+        total = st.f_blocks * st.f_frsize
+        used = (st.f_blocks - st.f_bfree) * st.f_frsize
+        return _ntuple_diskusage(total, used, free)
+
+elif os.name == 'nt':       # Windows
```

Comment:
Actually you can choose between these ones, so you have 2.7.13 if you want. For python 3 I think we have a few more challenges than science being slow..
 --> 2.5.4p2-i686-slc5-gcc43
 --> 2.5.4p2-x86_64-slc5-gcc43
 --> 2.6.5-i686-slc5-gcc43
 --> 2.6.5-x86_64-slc5-gcc43
 --> 2.6.5p1-i686-slc5-gcc43
 --> 2.6.5p1-x86_64-slc5-gcc43
 --> 2.7.10-x86_64-slc6-gcc49
 --> 2.7.10-x86_64-slc6-gcc62
 --> 2.7.13-x86_64-slc6-gcc49
 --> 2.7.13-x86_64-slc6-gcc62
 --> 2.7.3-i686-slc6-gcc47
 --> 2.7.3-x86_64-slc6-gcc47
 --> 2.7.4-x86_64-slc6-gcc48    current
 --> 2.7.9-x86_64-slc6-gcc48
 --> 2.7.9p1-x86_64-slc6-gcc49